Dockerfile

```diff
@@ -84,9 +110,16 @@ COPY nginx.conf /etc/nginx/nginx.conf
 # Install MeiliSearch
 RUN curl -L https://install.meilisearch.com | sh
 
+# Install Node 16 for internal dashboard server
+RUN curl -sL https://deb.nodesource.com/setup_16.x | sh
+RUN apt-get update
+RUN apt install nodejs
+
 ENV SERVING_PORT="8082"
 ENV SERVING_HOST="0.0.0.0"
 ENV ETCD_ARCH=""
 
 EXPOSE 7878
-CMD ["/usr/bin/supervisord"]
\ No newline at end of file
+EXPOSE 80
```

Comment:
changing the exposed port doesn't break anything?

Reply:
double checking

Reply:
it doesn't break anything that I've tested. we expose 7878 + 80 when we run FF in the CLI, this makes it explicit.